kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -135,8 +136,8 @@ define([
             },
             ui: {
                 buttons: {
-                    enabled: ['stop', 'expand'],
-                    disabled: ['play', 'top', 'bottom'],
+                    enabled: ['expand', 'stop', 'top', 'bottom'],
```

Comment:
reordered so they are in the same order as they appear in the interface... not that it makes any difference.